design-proposals/live-migration-for-bridged-pod-network.md

```diff
@@ -1,17 +1,15 @@
 # Overview
-This feature enables support of live-migrate for VMs with pod network in bridge and macvtap modes.
+This feature enables support of live-migrate for VMs with pod network in bridge mode.
 
 ## Motivation
-Masquerade and slirp are not so performant compared to bridge and macvtap modes.
+Masquerade and slirp are not so performant compared to bridge mode.
```

Comment:
To be honest, I never thought about performance as a reason to use bridge binding on the primary network.
If you need better performance, why not using a secondary network?
Can you please elaborate about the use case.

Reply:
We have no other options to expose VM as Kubernetes service.
We still need to communicate with the other pods in the cluster and reuse cloud load-balancers or MetalLB.
We forced to use pod network in any case we want to communicate with Kubernetes.

Reply:
Did you try to use masquerade with multiqueue?
That's the result of a comparison we did in the past (masquerade binding, 8vcpu 8multiqueue)-

Masquerade overhead is minimized with multiqueue.
Masquerade vs. bridge binding
TCP Stream: -37% (nomq) -13% (mq)
UDP Stream:  -2% (nomq) -6% (mq)
TCP Latency: -24% (nomq) -3% (mq)
UDP Latency: -6% (nomq) -2% (mq)

Reply:
@AlonaKaplan which tool you used to benchmark this?